portal/templates/utilisation-reporting-service/_partials/header.njk

```diff
@@ -0,0 +1,51 @@
+<header class="govuk-header " role="banner" data-module="govuk-header" data-cy="header">
```

Comment:
As mentioned this is more of a temporary addition until we look at the navigation ticket